### ISSUE TITLE
Is there any documentation somewhere?

### ISSUE THREAD

**Issue by thedrow** [benoitc/offset#22]

**Comment by ncoghlan** [comment]
Indeed, I'm trying to figure out how offset compares to goless (https://goless.readthedocs.org), but the lack of docs for offset makes it difficult to know where to start, or how it maps concepts from Go.

**Comment by benoitc** [comment]
@ncoghlan yeah doc is coming, i will release it next week with the upgrade, I was a bit busy with some due work last week ( as I planned to release my changes and write tests/docs). In the mean time you can have a look in the examples.

The major change that is coming is the removal of the `run` function. The scheduler will be triggered automatically.

**Comment by ncoghlan** [comment]
Great to hear! I have a syntax idea for how to make the goless select example easier to read, and if it works to make offset code easier to read as well, then it may actually be something worth doing for 3.5 :)
